lib/host.rb

```diff
@@ -827,4 +828,73 @@ def clean_up
       close
     end
   end
+
+  class OCDebugAccessibleHost < LinuxLikeHost
+    def initialize(hostname, opts={})
+      super
+      @workdir = "/host/tmp/workdir/" + EXECUTOR_NAME unless self[:workdir]
+      @exec_lock = Mutex.new
+    end
+
+    # @note execute commands without special setup
+    def exec_raw(*commands, **opts)
+      @accessed = true
+      unless opts[:single]
+        commands = ["bash", "-c", commands_to_string(commands)]
+      end
+
+      @exec_lock.synchronize {
+        # note this will block until timeout if command does not exist remotely
+        # TODO: check debug pod status in the background
+        # note2: exit status is always 0, WRKLDS-98
+        node.env.admin.cli_exec(
+          :debug,
+          resource: "node/#{node.name}",
+          oc_opts_end: "",
+          exec_command_arg: commands,
+          _stdin: opts[:stdin],
+          _stdout: opts[:stdout]
```

Comment:
Here need to verify that no spam will come into stdout

Reply:
chekced with 
```
➜  ~ oc version 
Client Version: v4.3.0
Server Version: 4.2.1
Kubernetes Version: v1.14.6+868bc38
```

And the prompt is in stderr, so it's ok here.
```
➜  ~ oc debug nodes/ip-10-0-133-47.us-east-2.compute.internal -- chroot /host ls /
Starting pod/ip-10-0-133-47us-east-2computeinternal-debug ...
To use host binaries, run `chroot /host`
bin
boot
dev
etc
home
lib
lib64
media
mnt
opt
ostree
proc
root
run
sbin
srv
sys
sysroot
tmp
usr
var

Removing debug pod ...
➜  ~ oc debug nodes/ip-10-0-133-47.us-east-2.compute.internal -- chroot /host ls / 2>/dev/null
bin
boot
dev
etc
home
lib
lib64
media
mnt
opt
ostree
proc
root
run
sbin
srv
sys
sysroot
tmp
usr
var
```

Reply:
So no concerns?

Reply:
yes